libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookAdapter.cs

```diff
@@ -46,19 +50,21 @@ public class FacebookAdapter : BotAdapter, IBotFrameworkHttpAdapter
         /// </remarks>
         /// <param name="logger">The logger this adapter should use.</param>
         public FacebookAdapter(IConfiguration configuration, ILogger logger = null)
-            : this(new FacebookClientWrapper(new FacebookAdapterOptions(configuration["FacebookVerifyToken"], configuration["FacebookAppSecret"], configuration["FacebookAccessToken"])), logger)
+            : this(new FacebookAdapterOptions(configuration["FacebookVerifyToken"], configuration["FacebookAppSecret"], configuration["FacebookAccessToken"]), logger)
         {
         }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="FacebookAdapter"/> class using an existing Facebook client.
         /// </summary>
-        /// <param name="facebookClient">A Facebook API interface.</param>
+        /// <param name="options">Options for the Facebook Adapter.</param>
         /// <param name="logger">The logger this adapter should use.</param>
-        /// <exception cref="ArgumentNullException"><paramref name="facebookClient"/> is null.</exception>
-        public FacebookAdapter(FacebookClientWrapper facebookClient, ILogger logger = null)
+        /// <param name="facebookClient">Client used to interact with the Facebook API.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="options"/> is null.</exception>
+        public FacebookAdapter(FacebookAdapterOptions options, ILogger logger = null, FacebookClientWrapper facebookClient = null)
```

Comment:
Does this break backward compatibility?

Reply:
Yes, but we are OK as this is a preview package so this is part of the pre GA changes.